README.md

```
@@ -1,2 +1,75 @@
+```
+
+### 2. Compile the project with CMake 
+```bash
 cmake -B .build && cmake --build .build
+```
+
+### 3. Run Raytracer
+```bash
+./raytracer ./scenes/example.cfg
+```
+## Features
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)